notebooks/oggm-edu/flowline_model.ipynb

```diff
@@ -27,26 +27,35 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# The commands below are just importing the necessary modules and functions\n",
-    "# Plot defaults\n",
+    "# import oggm-edu helper package\n",
+    "import oggm_edu as edu"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import modules, constants and set plotting defaults\n",
```

Comment:
What's the purpose of the changes in `flowline_model.ipynb` ?

Reply:
I created a new function to not repeat code and I improved some comments as I did the notebook myself.

Reply:
OK! I just wanted to know